func/code.func

```diff
@@ -28,34 +28,36 @@ cell remove_old_orders(cell orders, int o_seqno) {
   return orders;
 }
 
-_ pack_state(int o_seqno, int n_wins, int in_amount, int out_amount, int hash, slice owner1, slice owner2, slice echo, cell orders) inline {
+_ pack_state(int o_seqno, int n_wins, int in_amount, int out_amount, int hash, slice owner, slice ecvrf, slice player, int rand_updated, int player_coins, cell orders) inline {
   return begin_cell()             ;; FIRST CELL
     .store_uint(o_seqno, 64)      ;; 64 bit  | order seqno
     .store_uint(n_wins, 64)       ;; 64 bit  | number of wins
     .store_uint(in_amount, 256)   ;; 256 bit | ingoing amount
     .store_uint(out_amount, 256)  ;; 256 bit | outgoing amount
     .store_uint(hash, 256)        ;; 256 bit | seed salt
+    .store_uint(rand_updated, 32) ;; 32 bit  | bool rand_updated
```

Comment:
Can be replaced with `.store_bit(rand_updated)`, if `rand_updated` is always 0/-1.

Should also be renamed to `random_locked`.

Reply:
I couldn't find anything for store_bit, can you plz point similar function in doc?

Reply:
I may be mistaken but I certainly remember that this function exists. It may be embedded in FunC compiler, actually.

Reply:
Actually -1 takes 257 bits so we can have boolean as 0/1 with 8 bit int size

Reply:
> Actually -1 takes 257 bits

It actually can be stored in smaller storage, and 1 bit is sufficient.
0/1 -> `.store_uint(random_locked, 1)`
0/-1 -> `.store_int(random_locked, 1)`, equivalent to `.store_bit(random_locked)`.